src/test/java/com/puppycrawl/tools/checkstyle/checks/modifier/ModifierOrderCheckTest.java

```diff
@@ -68,7 +68,7 @@ public void testIt() throws Exception {
             "49:35: " + getCheckMessage(MSG_ANNOTATION_ORDER, "@MyAnnotation4"),
             "157:14: " + getCheckMessage(MSG_MODIFIER_ORDER, "default"),
         };
-        verify(checkConfig, getPath("InputModifier.java"), expected);
+        verify(checkConfig, getPath("InputModifierIt.java"), expected);
```

Comment:
Test is named `ModifierOrderCheckTest` so files must start with `InputModifierOrder`.
Please change.
All inputs for this class should follow the pattern. I see others aren't.
Class names of inputs should also follow pattern of file name.